proposals/lip-0050.md

```diff
@@ -34,27 +34,31 @@ Implementing the Legacy module avoids the need for other modules (specifically t
-| `MODULE_ID_LEGACY`             | bytes | TBD    | Module ID of the Legacy module.                                                                            |
-| `COMMAND_ID_RECLAIM`           | bytes | 0x0000 | Command ID of the reclaim command.                                                                         |
-| `COMMAND_ID_REGISTER_KEYS`     | bytes | 0x0001 | Command ID of the register keys command.                                                                   |
-| `TYPE_ID_ACCOUNT_RECLAIM`      | bytes | 0x0001 | The type ID of the account reclaim event.                                                                  |
-| `TYPE_ID_KEYS_REGISTERED`      | bytes | 0x0002 | The type ID of the keys registered event.                                                                  |
-| `STORE_PREFIX_LEGACY_ACCOUNTS` | bytes | 0x0000 | Store prefix of the legacy accounts substore. This contains the addresses and balances of legacy accounts. |
-| `LOCAL_ID_LSK`                 | bytes | 0x00000000 | Local identifier of the LSK token.                                                                     |
-| `TOKEN_ID_LSK_MAINCHAIN`       | bytes | 8 bytes all set to `0`  | Identifier of the LSK token on the Lisk mainchain.                                        |
-| `INVALID_BLS_KEY`              | bytes | 48 bytes all set to `0` | The BLS public key set during the migration for validators without a BLS key.             |
-| `INVALID_ED25519_KEY`          | bytes | 32 bytes all set to `255` | The Ed25519 public key set during the migration for validators without a generator key. |
-| `ADDRESS_LEGACY_RESERVE`       | bytes | `SHA-256(b'legacyReserve')[:20]` | The address used to store all tokens of legacy accounts.                         |
+| `MODULE_NAME_LEGACY`           | string|"legacy"| The name of the Legacy module.                                                                            |
+| `COMMAND_RECLAIM`              | string|"reclaim"| The name of the reclaim command.                                                                         |
+| `COMMAND_REGISTER_KEYS`        | string| "registerKeys" | The name of the register keys command.                                                                   |
+| `EVENT_NAME_ACCOUNT_RECLAIM`   | string | "accountReclaim" | The name of the account reclaim event.                                                                  |
+| `EVENT_NAME_KEYS_REGISTERED`   | string | "keysRegistered" | The name of the keys registered event.                                                                  |
+| `SUBSTORE_PREFIX_LEGACY_ACCOUNTS` | bytes | 0x0000 | Substore prefix of the legacy accounts substore. This contains the addresses and balances of legacy accounts. |
+| `TOKEN_ID_LSK`       | bytes | 0x0000000000000000  | Identifier of the LSK token.                                        |
+| `NUM_BYTES_ADDRESS`            | uint32| 20   | The length of an address in bytes.    |
```

Comment:
I would follow the convention `LENGTH_ADDRESS`, same for legacy address

Reply:
Ok, changed.